pulsar-proxy/src/test/java/org/apache/pulsar/proxy/server/ProxyConfigurationTest.java

```diff
@@ -53,5 +59,55 @@ public void testConfigFileDefaults() throws Exception {
         }
     }
 
+    @Test
+    public void testBackwardCompatibility() throws IOException {
```

Comment:
I want to keep testBackwardCompatibility(), if we have other changes need to keep backward compatibility, we can just add items to the test, otherwise, we need many tests methods.